<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="8">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Selectionner un Spectacle&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="9">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Selectionner</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="8">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ajouter une représentation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="9">
    <widget class="QComboBox" name="comboBox_spec"/>
   </item>
   <item row="3" column="0" colspan="9">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="9">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Date:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Selectionner une représentation :</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="9">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QComboBox" name="comboBox_MD"/>
   </item>
   <item row="14" column="8">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="8">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Modifier une représentation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="8">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Supprimer une représentation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Selectionner un date :</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1" colspan="7">
    <widget class="QComboBox" name="comboBox_SD"/>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Selectionner une promotion :</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1" colspan="7">
    <widget class="QComboBox" name="comboBox_SP"/>
   </item>
   <item row="14" column="0">
    <widget class="QLineEdit" name="promoRepM"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Promotion à appliqué  :</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Modfifier promotion</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="6">
    <widget class="QComboBox" name="comboBox_MP"/>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Puis saisisser l'attributs à modifer :</string>
     </property>
    </widget>
   </item>
   <item row="14" column="7">
    <widget class="QComboBox" name="comboBox_minute_M">
     <item>
      <property name="text">
       <string>00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>35</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>45</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>55</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="21" column="1" colspan="7">
    <widget class="QPushButton" name="But_Supprimer">
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0" colspan="9">
    <widget class="QLabel" name="label_gerer">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QComboBox" name="comboBox_minute">
     <item>
      <property name="text">
       <string>00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>35</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>45</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>55</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QComboBox" name="comboBox_heure">
     <item>
      <property name="text">
       <string>00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QComboBox" name="comboBox_annee">
     <item>
      <property name="text">
       <string>2021</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2022</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2023</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2025</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2026</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2027</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2028</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2029</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2030</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QComboBox" name="comboBox_mois">
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QComboBox" name="comboBox_jour">
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="6">
    <widget class="QComboBox" name="comboBox_heure_M">
     <item>
      <property name="text">
       <string>00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="5">
    <widget class="QComboBox" name="comboBox_annee_M">
     <item>
      <property name="text">
       <string>2021</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2022</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2023</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2025</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2026</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2027</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2028</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2029</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2030</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="4">
    <widget class="QComboBox" name="comboBox_mois_M">
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QComboBox" name="comboBox_jour_M">
     <property name="maximumSize">
      <size>
       <width>201</width>
       <height>22</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Date de la Représentation :</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="5">
    <widget class="QLineEdit" name="promoRepI"/>
   </item>
   <item row="7" column="3" colspan="5">
    <widget class="QPushButton" name="But_Inserer">
     <property name="text">
      <string>Inserer</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Promo :</string>
     </property>
    </widget>
   </item>
   <item row="15" column="3" colspan="5">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Modifier date</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>sel_spec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>modifier_dateRep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_SD</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>refresh_promoRep_supp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>But_Supprimer</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>supprimer_rep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>But_Inserer</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ajouter_rep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_MD</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>refresh_promoRep_mod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sel_spec()</slot>
  <slot>ajouter_rep()</slot>
  <slot>modifier_dateRep()</slot>
  <slot>supprimer_rep()</slot>
  <slot>modifier_promoRep()</slot>
  <slot>refresh_promoRep_mod()</slot>
  <slot>refresh_promoRep_supp()</slot>
 </slots>
</ui>
